pkg/txhandler-factory/simple/policyloop.go

```diff
@@ -0,0 +1,411 @@
+// Copyright © 2023 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simple
```

Comment:
The directory structure for this package looks wrong to me:

```
pkg/txhandler-factory/simple/policyloop.go
             ^^^^^^^^
```

This file is not a `factory`, but an implementation.

I would expect:
```
pkg/txhandler/simple/policyloop.go
````

If we need a separate factory package, that can be:
```
pkg/txhandler/factory
````